### PR TITLE
UIU-1778 Departments: Create UI permissions for Settings>Users>Departments CRUD

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
[[UIU-1778]](https://issues.folio.org/browse/UIU-1778) Departments: Create UI permissions for Settings>Users>Departments CRUD

**REQUIREMENT: Create the UI permissions:**
Settings (Users) : Create, edit, view, and delete departments
Settings (Users) : Create, edit, and view departments
Settings (Users) : View departments

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -657,6 +657,59 @@
           "configuration.entries.item.put"
         ],
         "visible": true
+      },
+      {
+        "permissionName": "ui-users.settings.departments.view",
+        "displayName": "Settings (Users): View departments",
+        "subPermissions": [
+          "settings.users.enabled"
```
Comment (BogdanDenis) [inline package.json:665]: You need to also add permissions that are defined by back-end. You can find permissions for Users app here https://github.com/folio-org/mod-users/blob/master/descriptors/ModuleDescriptor-template.json#L118

Basically, back-end defines which permissions are required to access each endpoint, and in the UI we're using these permissions to group them according to business needs

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: translations/ui-users/en.json [hunk 2]
```diff
@@ -781,6 +781,12 @@
   "permission.viewperms": "Users: Can view permissions assigned to users",
   "permission.viewproxies": "Users: Can view proxies assigned to users",
   "permission.viewuserservicepoints": "Users: Can view service points assigned to users",
+  "permission.settings.departments.view":"Settings (Users): View departments",
+  "permission.settings.departments.edit":"Settings (Users): Edit departments",
+  "permission.settings.departments.create":"Settings (Users): Create departments",
+  "permission.settings.departments.delete":"Settings (Users): Delete departments",
+  "permission.settings.departments.create.edit.view.delete":"Settings (Users): Create, edit, view, and delete departments",
```
Comment (Vladyslav-Velytskyi) [inline translations/ui-users/en.json:788]: As in other translations before
```suggestion
  "permission.settings.departments.all":"Settings (Users): Create, edit, view, and delete departments",
```

File: package.json [hunk 3]
```diff
@@ -657,6 +657,64 @@
...
+      },
+      {
+        "permissionName": "ui-users.settings.departments.create.edit.view",
+        "displayName": "Settings (Users): Create, edit, and view departments",
+        "subPermissions": [
+          "ui-users.settings.departments.create",
+          "ui-users.settings.departments.edit",
+          "ui-users.settings.departments.view"
+        ],
+        "visible": true
```
Comment (Vladyslav-Velytskyi) [inline package.json:717]: Better to change direction of this two objects
```suggestion
      {
        "permissionName": "ui-users.settings.departments.create.edit.view",
        "displayName": "Settings (Users): Create, edit, and view departments",
        "subPermissions": [
          "ui-users.settings.departments.create",
          "ui-users.settings.departments.edit",
          "ui-users.settings.departments.view"
        ],
        "visible": true
      },
      {
        "permissionName": "ui-users.settings.departments.all",
        "displayName": "Settings (Users): Create, edit, view, and delete departments",
        "subPermissions": [
          "ui-users.settings.departments.create.edit.view",
          "ui-users.settings.departments.delete"
        ],
        "visible": true
```

**Review by Vladyslav-Velytskyi — COMMENTED** [review]
Please, update CHANGELOG.md

**Pull request merged**